testlapack/dgetrs.go

```diff
@@ -0,0 +1,116 @@
+package testlapack
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/floats"
+)
+
+type Dgetrser interface {
+	Dgetrfer
+	Dgetrs(trans blas.Transpose, n, nrhs int, a []float64, lda int, ipiv []int, b []float64, ldb int)
+}
+
+func DgetrsTest(t *testing.T, impl Dgetrser) {
+	for _, test := range []struct {
+		trans             blas.Transpose
+		n, nrhs, lda, ldb int
+		tol               float64
+	}{
+		{blas.NoTrans, 3, 3, 0, 0, 1e-14},
+		{blas.Trans, 3, 3, 0, 0, 1e-14},
+		{blas.NoTrans, 3, 5, 0, 0, 1e-14},
+		{blas.Trans, 3, 5, 0, 0, 1e-14},
+		{blas.NoTrans, 5, 3, 0, 0, 1e-14},
+		{blas.Trans, 5, 3, 0, 0, 1e-14},
+
+		{blas.NoTrans, 3, 3, 8, 10, 1e-14},
+		{blas.Trans, 3, 3, 8, 10, 1e-14},
+		{blas.NoTrans, 3, 5, 8, 10, 1e-14},
+		{blas.Trans, 3, 5, 8, 10, 1e-14},
+		{blas.NoTrans, 5, 3, 8, 10, 1e-14},
+		{blas.Trans, 5, 3, 8, 10, 1e-14},
+
+		{blas.NoTrans, 300, 300, 0, 0, 1e-10},
+		{blas.Trans, 300, 300, 0, 0, 1e-10},
+		{blas.NoTrans, 300, 500, 0, 0, 1e-10},
+		{blas.Trans, 300, 500, 0, 0, 1e-10},
+		{blas.NoTrans, 500, 300, 0, 0, 1e-10},
+		{blas.Trans, 500, 300, 0, 0, 1e-10},
+
+		{blas.NoTrans, 300, 300, 700, 600, 1e-10},
+		{blas.Trans, 300, 300, 700, 600, 1e-10},
+		{blas.NoTrans, 300, 500, 700, 600, 1e-10},
+		{blas.Trans, 300, 500, 700, 600, 1e-10},
+		{blas.NoTrans, 500, 300, 700, 600, 1e-10},
+		{blas.Trans, 500, 300, 700, 600, 1e-10},
+	} {
```

Comment:
`for _, trans := range []blas.Transpose{blas.NoTrans, blas.Trans} { ... }`

and remove the blas.Transpose field.